### DISCUSSION TITLE
bash: adonis: command not found

### DISCUSSION THREAD

**Discussion by soreach** [adonisjs/core#1888] [Help]
I tried install adonjs with  npm i -g @adonisjs/cli ,but when I start create  adonis new my-app it show this error.

**Comment by ItsJuniorDias** [comment]
you can try to run the command:

`npm i --global @adonisjs/cli`
or
`yarn install --global @adonisjs/cli`

**Comment by tomgobich** [comment]
Are you attempting to create a new AdonisJS 5 application? If so, please read through the installation documentation. As shown in there, the command to create a new Adonis application is:
```bash
npm init adonis-ts-app@latest project-name
```
https://docs.adonisjs.com/guides/installation
